scripts/ci/pipeline-netcore-runtime.yml

```diff
@@ -4,7 +4,15 @@ pool:
 variables:
 - ${{ if ne(variables['System.TeamProject'], 'public') }}:
   - group: DotNet-VSTS-Bot
-
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), ne(variables['Build.Reason'], 'PullRequest')) }}:
+    # Variables used to publish packages to blob feed
+    - name: dotnetfeedUrl
+      value: https://dotnetfeed.blob.core.windows.net/dotnet-coreclr/index.json
+    - name: dotnetfeedPAT
```

Comment:
nit: I understand casing in most of the yml file examples isn't consistent, but it'd be nice to move towards some consistency here.  ie, I see things named with all kinds of casing / naming patterns...

- dotnetfeedUrl (yaml specific variable)
- dotnetfeedPAT (yaml specific variable but with different casing)
- dotnetfeed-storage-access-key-1 (different casing again - a victim of the variable name in key vault)
- _DotNetPublishToBlobFeed (different casing again - a victim of what the Arcade Sdk expects)

I guess my nit is really, can we change this to `dotnetFeedPat` and `dotnetFeedUrl`?